cloudformation/mongo-opsmanager-backup.template

```diff
@@ -346,7 +351,7 @@
             "IpProtocol": "tcp",
             "FromPort": "22",
             "ToPort": "22",
-            "CidrIp": "10.0.0.0/8"
+            "CidrIp": { "Ref": "SSHAccessCIDR" }
```

Comment:
I've made the default for `SSHAccessCIDR` 10.249.0.0/16 across all Mongo related templates.